README.md

```diff
@@ -1,38 +1,145 @@
-# Filter Employee Modal
+# Zadanie Rekrutacyjne - Michał Skrzypczyński
 
-## Description
+Plusy:
+- Wersja live
+- Readme
+- Fajne użycie styled-components z propsami
+- Proptypes
+- Custom hooks, np. z wielkością okna
 
-A simple website that filters employees by filters selected by a user.
-Made for recruitment purposes.
 
-## Preview
+Minusy:
+- config z pracownikami w App.js ?
+- Zero testów
```

Comment:
W specyfikacji zadania rekrutacyjnego nie było wzmianki o pisaniu testów. Z pewnością bym je zaimplementował, gdyby było inaczej. Z drugiej strony, pisałeś że oczekujecie jakości produkcyjnej, więc mogłem się prawdopodobnie domyśleć ;)

Reply:
To takie podchwytliwe , nie będę specjalnie tego ukrywał. Dla nas jakość produkcyjna jest równoznaczna z pokryciem testami. Obecnie mamy takie zasady, że nic na produkcje nie może trafić bez napisania testów. Ja często (szczególnie przy tworzeniu mikroserwisów) używam metodyki Test Driven Development. Wbrew pozorom oszczędzasz mnóstwo czasu , ktore zwykle schodzi na debugowanie, odpalanie aplikacji i testowanie. 

Niestety mieliśmy fazę gwałtownego rozwoju kilka lat temu jako startup i wtedy zaniedbaliśmy zasady z testami -> i do dzisiaj to nadrabiamy :)